### PR TITLE
Update NEXUS_Config.rc

### PR DESCRIPTION
**Pull request by bbakernoaa**
re-enable Megan in config/cmaq

### CODE REVIEW

**Comment by bbakernoaa** [comment]
@zmoon

**Comment by zmoon** [comment]
Is this other setting not needed? (from when you turned it off) https://github.com/noaa-oar-arl/NEXUS/pull/17/commits/756418d016df4474e66820707395c679159ec893

**Comment by bbakernoaa** [comment]
thanks for catching that @zmoon

File: config/cmaq/NEXUS_Config.rc [hunk 1]
```diff
@@ -132,7 +132,7 @@ Cap time shift              : true
     --> +LinStratChem+    :       false
     --> CEDS              :  on   true
     --> OMI_HTAP_SO2      :  on   true
-    --> Yuan_XLAI         :  off  false
+    --> Yuan_XLAI         :  on  false
```
Comment (zmoon) [inline config/cmaq/NEXUS_Config.rc:135]: prettier
```suggestion
    --> Yuan_XLAI         :  on   false
```

**Pull request merged**